### ISSUE TITLE
F824 erroneously detects usage of global lists

### ISSUE THREAD

**Issue by hmaarrfk** [PyCQA/flake8#1977]
### how did you install flake8?

```console
conda install flake8

through conda-forge

Should i open this on https://github.com/PyCQA/pyflakes ???
```

### unmodified output of `flake8 --bug-report`

```json
{
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.12.8",
    "system": "Linux"
  },
  "plugins": [
    {
      "plugin": "mccabe",
      "version": "0.7.0"
    },
    {
      "plugin": "pycodestyle",
      "version": "2.12.1"
    },
    {
      "plugin": "pyflakes",
      "version": "3.3.2"
    }
  ],
  "version": "7.2.0"
}
```

#### what I expected to happen

Usage of global lists triggers F824 error. One might not want to assign them, but they are used with `my_list.append` and thus this should not trigger an error.

#### sample code

```python

this = []


def that():
    global this
    this.append(1)


print(this)
that()
print(this)
that()
print(this)
```

#### commands ran

```console
$ flake8 foo,py

foo.py:6:5: F824 `global this` is unused: name is never assigned in scope
```
